databases/postgres_db.js

```diff
@@ -24,6 +24,9 @@ exports.Database = class extends AbstractDatabase {
     super();
     this.settings = settings;
 
+    // Support Strings such as "postgres://etherpad:etherpad@localhost:5432/etherpad"
+    if (typeof this.settings === 'string') this.settings = new URL(this.settings);
```

Comment:
This won't work because `pg.Pool()` expects a config object, not a URL object. Here's a fix:

```suggestion
    if (typeof this.settings === 'string') this.settings = {connectionString: this.settings};
```